scripts/exportAssets.py

```diff
@@ -1,44 +1,87 @@
 #!/usr/bin/python
 import os
+import requests
 import sys
 import argparse
 import json
-from util import ElasticsearchUtil
-from util import LogConfig
 from util import Utility
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
 
 EXPORT_LOG = "/tmp/ExportAssets.log"
-
-esUtil = ElasticsearchUtil(EXPORT_LOG)
-logger = LogConfig(log_file=EXPORT_LOG)
-logging, rotating_handler = logger.configure_and_return_logging()
 UTIL = Utility(log_file=EXPORT_LOG)
+logging, rotating_handler = UTIL.get_logging()
+logger = logging.getLogger()
 
 OUTPUT_DIR = os.path.dirname(os.path.realpath(__file__)) 
 
-INDEX = esUtil.KIBANA_INDEX
 TYPE = None
-ID = None
+TITLE = None
 
 DASHBOARD = "dashboard"
 VISUALIZATION = "visualization"
 SEARCH = "search"
 
 FILE_PATHS_AND_CONTENTS = {}
 
-
-def get_asset(es_index, es_type, es_id):
-    ignored, doc_exists = esUtil.function_with_timeout(esUtil.ES_QUERY_TIMEOUT,
-                                                       esUtil.document_exists,
-                                                       es_index,
-                                                       es_type,
-                                                       es_id)
-    if doc_exists:
-        success, ret = esUtil.get_document(es_index, es_type, es_id)
-        logging.info("GET RETURNS: " + "\n" + UTIL.pretty_format(ret))
-        return True, ret
-    else:
-        logging.info("No such thing as " + es_id + " in /" + es_index + "/" + es_type + "/")
+def check_elasticsearch_health():
+    try:
+        url = 'http://localhost:9200/_cluster/health?wait_for_status=yellow&timeout=60s'
+        session = requests.Session()
+        response = session.get(url)
+        if not response.ok:
+            response.raise_for_status()
+
+        logger.info("elasticsearch status: " + "\n" + UTIL.pretty_format(response.json()))
+        return True
+
+    except Exception as err:
+        logger.warning("[loadAssets.py] Caught HTTP exception: {0}".format(err))
+        return False
+
+def get_search_title_json(es_type, es_title):
+    esField = es_type + '.title'
```

Comment:
Guido Van Rossum frowns upon your camel case, python PEP8 prefers snake case for functions and variables http://www.python.org/dev/peps/pep-0008/